engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/DefinitionExtractor.scala

```diff
@@ -38,7 +36,7 @@ class DefinitionExtractor[T](methodDefinitionExtractor: MethodDefinitionExtracto
 
   private def findMethodToInvoke(obj: Any): Method = {
     val methodsToInvoke = obj.getClass.getMethods.toList.filter { m =>
-      m.getAnnotation(classOf[MethodToInvoke]) != null
+      m.getAnnotation(classOf[MethodToInvoke]) != null || hasContextTransformationAnnotation(m)
```

Comment:
Why? @MethodToInvoke can be used to make stuff consistent...
*OR* we can add sth as parameter to @MethodToInvoke?